src/components/network/NewNetwork.tsx

```diff
@@ -78,16 +81,22 @@ const NewNetwork: React.SFC = () => {
             eclairNodes: 1,
             bitcoindNodes: 1,
             customNodes: initialCustomValues,
+            externalNetworkName: '',
           }}
           onFinish={createAsync.execute}
         >
-          <Form.Item
-            name="name"
-            label={l('nameLabel')}
-            rules={[{ required: true, message: l('cmps.forms.required') }]}
-          >
-            <Input placeholder={l('namePhldr')} />
-          </Form.Item>
+          <Col>
+            <Form.Item
+              name="name"
+              label={l('nameLabel')}
+              rules={[{ required: true, message: l('cmps.forms.required') }]}
+            >
+              <Input placeholder={l('namePhldr')} />
+            </Form.Item>
+            <NewNetworkOptions
+              setIsDockerNetworkNameValid={setIsDockerNetworkNameValid}
+            />
```

Comment:
I'm not really a big fan of how this looks on the screen. I would definitely place this at the bottom of the form, right above the button.